e2e-tests/commands/shared-commands/assertions/assert-loss-payee-financial-uk-field-values.js

```diff
@@ -0,0 +1,34 @@
+import { field as fieldSelector, autoCompleteField } from '../../../pages/shared';
+import { POLICY as POLICY_FIELD_IDS } from '../../../constants/field-ids/insurance/policy';
+import mockApplication from '../../../fixtures/application';
+
+const { LOSS_PAYEE_FINANCIAL_UK } = mockApplication.POLICY;
+
+const {
+  LOSS_PAYEE_FINANCIAL_UK: {
+    ACCOUNT_NUMBER, SORT_CODE,
+  },
+  FINANCIAL_ADDRESS,
+} = POLICY_FIELD_IDS;
+
+/**
+ * assertLossPayeeFinancialUkFieldValues
+ * Assert all field values in the "loss payee financial uk" form.
+ * @param {String} expectedAccountNumber: Account number
+ * @param {String} expectedSortCode: Sort code
+ * @param {String} expectedBankAddress: Bank address
+ */
+const assertLossPayeeFinancialUkFieldValues = ({
+  expectedAccountNumber = LOSS_PAYEE_FINANCIAL_UK[ACCOUNT_NUMBER],
+  expectedSortCode = LOSS_PAYEE_FINANCIAL_UK[SORT_CODE],
+  expectedBankAddress = LOSS_PAYEE_FINANCIAL_UK[FINANCIAL_ADDRESS],
```

Comment:
Should this have `Financial` instead of `Bank`, to be aligned with other naming?